src/test/ArraysTable.fbs

```diff
@@ -0,0 +1,53 @@
+struct Point3b {
```

Comment:
Good thing to do might be to set up some structs that could have weird alignment. E.g.

```
struct HasGaps {
  small:ubyte;
  // If I understand things right, there should be a 7 byte gap for padding here.
  big:[int64:3];
}
```